generators/spring-boot/index.cjs

```diff
@@ -0,0 +1,153 @@
+/**
```

Comment:
hosted in generators/backend/spring-boot?

Reply:
Same as generators/build-tool/maven, I am avoiding sub-sub-generators for now.